required/tools/longtable.dtx

```diff
@@ -1066,7 +1081,42 @@
 % so that "\caption" works correctly.
 %    \begin{macrocode}
 \def\LT@array[#1]#2{%
-  \refstepcounter{table}\stepcounter{LT@tables}%
+%    \end{macrocode}
+%
+%    With respect to tagging we have a complicated situation with
+%    longtable. When at the begin the \cs{endhead},
+%    \cs{endfirsthead}, \cs{endfoot} and \cs{endlastfoot} are used to
+%    setup head and foot they create each a structure subtree with one or
+%    more rows. From this structures we want to keep at most two (head
+%    and foot) and move the foot to the end of the table. When the head
+%    and foot boxes are (re)inserted on following pages we want to mark
+%    them up as artifact with the exception of the head at the begin and
+%    the foot box at the end.
+%
+%    TODO: When a line is killed the structure subtree is there already
```

Comment:
yes, maybe , but it is probably not too hard for @u-fischer ... we'll see. I added a further note.

Reply:
hm, not easy. Currently \kill (with table-tagging) errors, and if I avoid the error it doesn't kill the row (don't know why). And if that problem can be resolved: getting tagging right is not trivial either, it would probably require some label and an additional counter to keep track on the rows that should be ignored. I'm inclined to make that a simply error with "sorry not supported with tagging".

Reply:
I guess part of the problem is that it acts "after the fact" so to speak. That means one would basically either wait with a lot of things to see what comes at the end of a row or maybe record the fact that there was a kill in row X and next time around do the killing  properly.
Instead if this we could perhaps have a \killstart command to be added at the beginning of the row which would tell the tagging what's up. That would still mean some manual work with such documents but at least the functionality could be preserved.

Perhaps @davidcarlisle  has some thoughts?

In a first version, just stating that this doesn't work with tagging is certainly ok.

Reply:
I was going to suggest same, whether `\xkill ..... \\` would be easier so that knew up front that this was a killed row. Can you handle `\kill` in the kernel `tabbing` ?  (on which the longtable version was based)